styles.css

```diff
@@ -0,0 +1,35 @@
+/* === Generelle Einstellungen === */
+
+:root {
+--theme-color-1: #fbeefb; /* Hintergrund */
+--theme-color-2: #525252; /* Nav1 und Footer Hintergrund */
+--theme-color-3: #f3d5f3; /* Nav2 und Überschriften */
+--theme-color-4: #ffcffc; /* Divs (e.g. Produkt und Kategorie auf Main Page) */
+--theme-color-5: #ffe6fc; /* Andere Divs */
+--theme-color-6: #fcf4f4; /* Hintergrundfarbe für Input Felder*/
+--font-color: #0c0b0c; /* Schriftfarbe */
```

Comment:
Kommentare auf gleiche Ebene (per Tabs in den gleichen Abstand bringen)

Reply:
Is fixed